src/components/ProjectFilter/index.js

```diff
@@ -0,0 +1,22 @@
+import React, { useState } from "react";
+import Label from "../Label";
+import "./ProjectFilter.css";
+
+function ProjectFilter({filters}) {
+  const [selectedFilter, setSelectedFilter] = useState([]);
+  return (
+    <div>
+      {filters.map((filter) => (
+        <Label
+          text={filter}
+          onClick={() => setSelectedFilter(filter)}
```

Comment:
@AbdurrahmanSogbesan this onclick will be wrong. The `selectedFilter ` state is an array. so if you setSelected to one filter it becomes a text. Youre supoosed to add the filter to the list when clicked. So when setting state you cant set the filter. you gotta keep the old ones, and add the fulter. sth like`[...oldFilters, filter]` is what youre setting to state. Adjust for this and push this branch again